os/net/lwip/src/api/sockets.c

```diff
@@ -84,6 +84,7 @@
 #include <net/lwip/tcpip.h>
 #include <net/lwip/pbuf.h>
 #include <net/lwip/mem.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting htons related error to overcome that error we have added this header file.

Reply:
It is better to edit Make.defs and add the -Wno-implicit-function-declaration flag instead, than having to modify lwip source files in this pull request. Please revert and modify Make.defs as you find appropriate.